PowerUp/app/src/main/res/layout-land/gamemap.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/map"
+    >
+
+    <Button
+        android:id="@+id/HouseButton"
+        android:layout_width="@dimen/map_click_width"
+        android:layout_height="@dimen/map_click_height"
+        android:layout_marginLeft="57dp"
```

Comment:
Don't use raw values. Define them on dimens.xml and reference them here using `@dimen/dimension_name`